community/server/src/test/java/org/neo4j/server/rest/repr/DatabaseRepresentationTest.java

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright (c) 2002-2015 "Neo Technology,"
+ * Network Engine for Objects in Lund AB [http://neotechnology.com]
+ *
+ * This file is part of Neo4j.
+ *
+ * Neo4j is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+package org.neo4j.server.rest.repr;
+
+import org.junit.Test;
+
+import java.net.URI;
+import java.util.HashMap;
+
+import org.neo4j.server.rest.repr.formats.MapWrappingWriter;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.equalTo;
+import static org.mockito.Mockito.mock;
+import static org.neo4j.helpers.collection.MapUtil.map;
+
+public class DatabaseRepresentationTest
+{
+    @Test
+    public void shouldIncludeExpectedResourcePaths() throws Throwable
+    {
+        // Given
+        DatabaseRepresentation repr = new DatabaseRepresentation();
+
+        // When
+        HashMap<String,Object> output = new HashMap<>();
+        repr.serialize( new MappingSerializer( new MapWrappingWriter( output ), URI.create("http://steveformayor.org"), mock(ExtensionInjector.class) ) );
+
+        // Then
+        assertThat( output, equalTo( map(
+            "relationship_index", "http://steveformayor.org/index/relationship",
+            "node_index", "http://steveformayor.org/index/node",
+            "batch", "http://steveformayor.org/batch",
+            "constraints", "http://steveformayor.org/schema/constraint",
+            "node", "http://steveformayor.org/node",
+            "extensions_info", "http://steveformayor.org/ext",
+            "node_labels", "http://steveformayor.org/labels",
+            "indexes", "http://steveformayor.org/schema/index",
+            "neo4j_version", "dev",
```

Comment:
Your test is failing because this version is incorrect in CI. I think you should somehow ignore the version value and only assert that the map has the `neo4j_version` key.